### DISCUSSION TITLE
向量化的时候indexes: undefined是什么意思呢？

### DISCUSSION THREAD

**Discussion by Ethereal-sakura** [labring/FastGPT#962] [Q&A]
如图所示
![image](https://github.com/labring/FastGPT/assets/110333012/ec717217-a343-434b-94a3-214efdd59557)

**Comment by c121914yu** [comment]
> 如图所示
> ![image](https://github.com/labring/FastGPT/assets/110333012/ec717217-a343-434b-94a3-214efdd59557)
> 

没有自定义索引内容

**Reply by Ethereal-sakura** [reply]
您好，感谢回复，请问会有影响吗，数据库会自动生成索引吗，我的docker logs 里面只有部分数据会出现这个情况，出现这个情况的原因是什么，我采用的是直接传入txt文本的方式构造数据库

**Reply by c121914yu** [reply]
> 
> 您好，感谢回复，请问会有影响吗，数据库会自动生成索引吗，我的docker logs 里面只有部分数据会出现这个情况，出现这个情况的原因是什么，我采用的是直接传入txt文本的方式构造数据库

都会创建默认的索引
